enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
@mattdrayer Would you mind if I called this contact model work a separate ticket? I probably don't have enough time to work it through this week since I budgeted my time based on the ENT-213 description, and I think we can get the Ecommerce PR for ENT-213 merged this week since it doesn't actually have a hard dependency on this change. :)